docs/zkapps/tutorials/anonymous-message-board.mdx

```diff
@@ -82,8 +71,7 @@ Example smart contract files come with the new project, you can delete them if y
 1. Optional: Delete the example files not used in this tutorial:
 
   ```sh
-  $ rm src/Add.ts
-  $ rm src/Add.test.ts
+  $ rm -rf src/*
```

Comment:
Nice. We should probably use this pattern of removing everything in the src folder in all the tutorials unless the index file is used.

Reply:
I meant to communicate that I think we should use the pattern you have here `rm -rf src/*` in all the tutorials that don't use the index file.